src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,69 @@
+package mate.academy.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinemaHalls")
```

Comment:
```suggestion
@Table(name = "cinema_halls")
```
like this in every entity class